portfolio/src/main/java/com/google/sps/servlets/DateServlet.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.servlets;
+
+import java.io.IOException;
+import java.util.Date;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
```

Comment:
Be sure to update the comments when you change code -- this still says `/hello` instead of `/date`.